code/game/machinery/bots/mulebot.dm

```diff
@@ -160,10 +144,27 @@ var/global/mulebot_count = 0
 			user.visible_message("<span class='warning'>[user] knocks [load] off [src] with \the [I]!</span>", "<span class='warning'>You knock [load] off [src] with \the [I]!</span>")
 		else
 			to_chat(user, "You hit [src] with \the [I] but to no effect.")
-	else
-		..()
 	return
 
+/obj/machinery/bot/mulebot/emag(mob/user)
```

Comment:
There's some cleaning in here and beepsky's. Just make sure it doesn't break anything. I think this kind of breaks pAIs